node/src/chain_spec.rs

```diff
@@ -183,6 +183,11 @@ pub fn local_config() -> Result<ChainSpec, String> {
 }
 
 pub fn testnet_config() -> Result<ChainSpec, String> {
+    ChainSpec::from_json_bytes(&include_bytes!("../chain-specs/zkverify_testnet.json")[..])
+}
+
+#[allow(dead_code)]
+pub fn testnet_config_build() -> Result<ChainSpec, String> {
```

Comment:
Why don't use this function for `test-dev` chain name?

Reply:
Addressed as per your suggestion (just opted for `test_build` naming).